src/goto-symex/symex_assign.cpp

```diff
@@ -36,6 +36,14 @@ void goto_symext::symex_assign_rec(
   code_assignt deref_code=code;
 
   clean_expr(deref_code.lhs(), state, true);
+  // make the structure of the lhs as simple as possible to avoid,
```

Comment:
This comment may be better placed in the Purpose section function since this might not be the only place it would get called